FirebaseAI.podspec

```diff
@@ -0,0 +1,70 @@
+Pod::Spec.new do |s|
+  s.name             = 'FirebaseAI'
+  s.version          = '11.13.0'
+  s.summary          = 'Firebase AI SDK'
+
+  s.description      = <<-DESC
+Build AI-powered apps and features with the Gemini API using the Firebase AI SDK.
+                       DESC
+
+  s.homepage         = 'https://firebase.google.com'
+  s.license          = { :type => 'Apache-2.0', :file => 'LICENSE' }
+  s.authors          = 'Google, Inc.'
+
+  s.source           = {
+    :git => 'https://github.com/firebase/firebase-ios-sdk.git',
+    :tag => 'CocoaPods-' + s.version.to_s
+  }
+
+  s.social_media_url = 'https://twitter.com/Firebase'
+
+  ios_deployment_target = '15.0'
+  osx_deployment_target = '12.0'
+  tvos_deployment_target = '15.0'
+  watchos_deployment_target = '8.0'
+
+  s.ios.deployment_target = ios_deployment_target
+  s.osx.deployment_target = osx_deployment_target
+  s.tvos.deployment_target = tvos_deployment_target
+  s.watchos.deployment_target = watchos_deployment_target
+
+  s.cocoapods_version = '>= 1.12.0'
+  s.prefix_header_file = false
+
+  s.source_files = [
+    'FirebaseAI/Sources/**/*.swift',
+  ]
+
+  s.swift_version = '5.9'
```

Comment:
I'd be fine to start this one at 6.0.

Reply:
Good call. I'll try this in a separate PR.